DXClient.sln

```diff
@@ -174,13 +174,19 @@ Global
 		{97458C1E-2E6C-4C5C-93C7-16A6712802E9}.AresWindowsGLRelease|x86.ActiveCfg = AresWindowsGLRelease|x86
 		{97458C1E-2E6C-4C5C-93C7-16A6712802E9}.AresWindowsGLRelease|x86.Build.0 = AresWindowsGLRelease|x86
 		{97458C1E-2E6C-4C5C-93C7-16A6712802E9}.AresWindowsXNADebug|Any CPU.ActiveCfg = AresWindowsXNADebug|Any CPU
+		{97458C1E-2E6C-4C5C-93C7-16A6712802E9}.AresWindowsXNADebug|Any CPU.Build.0 = AresWindowsXNADebug|Any CPU
```

Comment:
I remember XNA build is only supported as a x86 build. Not pretty sure.

Reply:
My bad. I will fixed that.

Reply:
It seems that Visual Studio automatically add these lines